### PR TITLE
Feat/fetch and pass ai tech data

### PR DESCRIPTION
**Pull request by alfonsojack**
### Did this break anything?
- [x]  No
- [ ]  Yes
### Type of change
- [x]  New feature (non-breaking change which adds functionality)
- [x]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [x]  Skip all the other stuff and briefly explain the fix.
### Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [ ]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [ ]  The code is DRY. If it's not, I tried my best
- [ ]  I reviewed my code before pushing
### Summary:
- File(s) added/changed: ----
- Short description of changes ---- Set up API call and correctly passed props then styled the form to get the page working well. Fixed most bugs and edited comment form to be roughly formatted the way we want

### CODE REVIEW

File: src/assets/background1.jpeg
Comment (Nicolelam8891) [inline src/assets/background1.jpeg:1]: This picture matches our splash page so much better! Great work finding this and using it as the background.

**Review by jmcsorle — APPROVED** [review]
This is such great work, Jack. Thank you for doing all of this!

File: src/App.js [hunk 1]
```diff
@@ -21,9 +21,18 @@ const App = () => {
...
+  console.log('afsdfas', selectedTechParam)
+
   const handleTechParamsSelection = (category, techParam) => {
     setSelectedCategory(category);
     setSelectedTechParam(techParam);
+    const techArr = [];
+    techArr.push(selectedTechParam);
+    const disabilityArr = [];
+    disabilityArr.push(needs[category]['disability parameter'])
+    console.log('disarr', disabilityArr)
```
Comment (jeremiahblackol) [inline src/App.js]: I already know you will, just a reminder to get rid of console.logs.

File: src/components/CategoryCard.css [hunk 2]
```diff
@@ -1,8 +1,14 @@
-.category-card {
+/* .category-card {
   box-sizing: border-box;
   border: 3px solid black;
   padding: 10px;
   height: 40vh;
   aspect-ratio: 1/1;
   margin: 10px;
+  background-color: hsl(125, 65%, 93%);
```
[the remark this answers is not in the document]
Comment (jeremiahblackol) [reply src/components/CategoryCard.css:8]: Y'all are doing a really good job w/ styling and considering accessibility.

**Pull request merged**